price_rule.go

```diff
@@ -84,6 +84,61 @@ type PriceRulesResource struct {
 	PriceRules []PriceRule `json:"price_rules"`
 }
 
+func (pr *PriceRule) SetPrerequisiteSubtotalRange(greaterThanOrEqualTo *string) {
+	if greaterThanOrEqualTo == nil {
+		pr.PrerequisiteSubtotalRange = nil
+	} else {
+		pr.PrerequisiteSubtotalRange = &prerequisiteSubtotalRange{
+			GreaterThanOrEqualTo: *greaterThanOrEqualTo,
+		}
+	}
+}
+
+func (pr *PriceRule) SetPrerequisiteQuantityRange(greaterThanOrEqualTo *int) {
+	if greaterThanOrEqualTo == nil {
+		pr.PrerequisiteQuantityRange = nil
+	} else {
+		pr.PrerequisiteQuantityRange = &prerequisiteQuantityRange{
+			GreaterThanOrEqualTo: *greaterThanOrEqualTo,
+		}
+	}
+}
+
+func (pr *PriceRule) SetPrerequisiteShippingPriceRange(lessThanOrEqualTo *string) {
```

Comment:
Do we know if lessThanOrEqual can be any string or just a money value?
If it is a money value - it will be good to either change the input type or to add an additional validation which prevents values to be set to what is not supported by Shopify

Reply:
Based on their [API documentation](https://shopify.dev/docs/admin-api/rest/reference/discounts/pricerule) I believe they accept money values in string form. Since the API itself is accepting strings I will add validation for this within each setter, ensuring that it can be read as money value.

Reply:
Confirmed via direct Shopify API call that they do validate decimal format for these values:

![image](https://user-images.githubusercontent.com/73915353/107804614-b1435a00-6d29-11eb-869c-dc6120de1173.png)

Reply:
Validation added as discussed.